src/deploy/campaign/environments.ts

```diff
@@ -117,8 +119,56 @@ export const getConfig = (
     rootPriceConfig: priceConfig,
     zeroVaultAddress: process.env.ZERO_VAULT_ADDRESS ? process.env.ZERO_VAULT_ADDRESS : zeroVault.address,
     mockMeowToken: process.env.MOCK_MEOW_TOKEN ? !!process.env.MOCK_MEOW_TOKEN : true,
-    stakingTokenAddress: process.env.STAKING_TOKEN_ADDRESS ? process.env.STAKING_TOKEN_ADDRESS : MEOW_TOKEN
+    stakingTokenAddress: process.env.STAKING_TOKEN_ADDRESS ? process.env.STAKING_TOKEN_ADDRESS : MEOW_TOKEN,
   };
 
+  // Will throw an error based on any invalid setup, given the `ENV_LEVEL` set
+  validate(config);
+
   return config;
 };
+
+// For testing the behaviour when we manipulate, we have an optional "env" string param
```

Comment:
this should be read from the ENV directly. the test should also reset this value in the ENV rather than passing here. the way it is now we are not really testing the environment setup, just that we passed the right argument to this function.